### PR TITLE
Refactor PageHeader

### PR DESCRIPTION
**Pull request by jfurrow**
Per @ashenden's request, the `PageHeader` will now vertically align the icon, text heading, and action buttons. The sub-heading is now left-aligned underneath all three of these elements.

Relevant integration tests passing:

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by rcorral** [comment]
restest this please

File: src/js/components/PageHeader.js [hunk 1]
```diff
@@ -129,7 +140,7 @@ PageHeader.propTypes = {
     React.PropTypes.string,
     React.PropTypes.object,
   ]),
-  pageHeaderHeadingClassNames: React.PropTypes.oneOfType([
+  pageHeaderContentWrapperClassNames: React.PropTypes.oneOfType([
```
Comment (mlunoe) [inline src/js/components/PageHeader.js]: This could also be an array

File: src/js/components/__tests__/ServiceInfo-test.js [hunk 2]
```diff
@@ -52,13 +52,13 @@ describe('ServiceInfo', function () {
 
     it('renders health state', function () {
       expect(
-        this.node.querySelectorAll('.page-header-text')[0].children[1].children[0].textContent
+        this.node.querySelectorAll('.page-header-sub-heading')[0].children[0].children[0].textContent
```
Comment (mlunoe) [inline src/js/components/__tests__/ServiceInfo-test.js]: #nit: If you are just selecting the first anyway, you can just use `querySelector` (i.e. no `All`)

**Comment by mlunoe** [comment]
Minor comments. Looks good! 🚢

File: src/js/components/PageHeader.js (hunk 1 above)
Comment (jfurrow) [inline src/js/components/PageHeader.js]: Alright I change all of them in this file.

File: src/js/components/__tests__/ServiceInfo-test.js (hunk 2 above)
Comment (jfurrow) [inline src/js/components/__tests__/ServiceInfo-test.js]: Alright I change all of them in this file.

**Comment by kennyt** [comment]
retest this please

**Comment by rcorral** [comment]
retest this please

**Pull request merged**